optuna/multi_objective/samplers/_nsga2.py

```diff
@@ -23,6 +25,7 @@
 _POPULATION_CACHE_KEY_PREFIX = "multi_objective:nsga2:population"
 
 
+@deprecated("2.3.0", "3.0.0")
 @experimental("1.5.0")
 class NSGAIIMultiObjectiveSampler(BaseMultiObjectiveSampler):
```

Comment:
More of a question, but is it difficult to make either NSGA-II sampler an alias? Might be easier to maintain.

Reply:
Looks good. I'm aiming to make the whole multi_objective module an alias. Thanks for the suggestion.